website/server/models/user/methods.js

```diff
@@ -20,10 +20,11 @@ schema.methods.getGroups = function getUserGroups () {
   return userGroups;
 };
 
-schema.methods.sendMessage = async function sendMessage (userToReceiveMessage, message) {
+schema.methods.sendMessage = async function sendMessage (userToReceiveMessage, translatedMessage, message) {
```

Comment:
Ah well, both translations would be the same in that case, so the point is moot.